src/test/java/org/broadinstitute/consent/http/service/UserServiceTest.java

```diff
@@ -78,7 +82,7 @@ public void setUp() {
     }
 
     private void initService() {
-        service = new UserService(userDAO, userPropertyDAO, userRoleDAO, voteDAO, institutionDAO, libraryCardDAO, samDAO);
+        service = new UserService(userDAO, userPropertyDAO, userRoleDAO, voteDAO, institutionDAO, libraryCardDAO, samDAO, jdbi);
```

Comment:
A disadvantage of this approach in the service class is that we can't (and shouldn't) have an integration test on the code block that does the multi-statement transaction.